Cargo.toml

```diff
@@ -43,3 +44,86 @@ derivable_impls = { level = "allow", priority = 2 }                  # false pos
 stable_sort_primitive = { level = "allow", priority = 2 }            # prefer stable sort
 extra-unused-type-parameters = { level = "allow", priority = 2 }     # stylistic
 default_constructed_unit_structs = { level = "allow", priority = 2 } # stylistic
+
+[workspace.dependencies]
+async-trait = { version = "0.1" }
+base58 = { version = "0.2", default-features = false }
+base64 = { version = "0.13", default-features = false, features = ["alloc"] }
+chrono = { version = "0.4", default-features = false, features = ["serde"] }
+clap = { version = "4.5", features = ["derive"] }
+log = { version = "0.4", default-features = false }
+scale-info = { version = "2.11", default-features = false, features = ["derive", "serde"] }
+codec = { package = "parity-scale-codec", version = "3.6", default-features = false, features = ["derive", "max-encoded-len"] }
+serde = { version = "1.0", default-features = false, features = ["derive", "alloc"] }
+serde_json = { version = "1.0", default-features = false }
+hex = { version = "0.4", default-features = false }
+hex-literal = { version = "0.4", default-features = false }
+futures = { version = "0.3.30", features = ["thread-pool"] }
+jsonrpsee = { version = "0.16", features = ["server"] }
+
+# substrate client
+sc-cli = { git = "https://github.com/paritytech/polkadot-sdk", tag = "polkadot-v1.3.0" }
```

Comment:
Sounds like a weird condition, but it is better to use, branch = "release-polkadot-v1.3.0". Since Frontier use the branch to follow its update. For 1.3.0, the tag and branch are weird to be different commit.